app/webutil.js

```diff
@@ -121,22 +121,22 @@ export function initSettings (callback /*, ...callbackArgs */) {
             }
         });
     } else {
-        // No-op
+        settings = {};
```

Comment:
Why is this added?

Reply:
Mostly for the tests so that the settings object can be reset between tests.  However, if `initSettings` were to be called again (which it isn't), it makes sense to reset the settings object.

Reply:
Ah, makes sense.